distribution/lib/Standard/Database/0.0.0-dev/src/Data/Table.enso

```diff
@@ -819,6 +819,77 @@ type Table
             on_problems.attach_problems_before problems <|
                 self.updated_context_and_columns new_ctx new_columns
 
+    ## Returns a new table with a chosen subset of columns left unchanged and
```

Comment:
If we are using `pivot` so much to call this `transpose` method, shall we maybe add `ALIAS pivot`?

Reply:
Will add to the design and update as part of the next task.